util/logtools/split_log.js

```diff
@@ -2,13 +2,14 @@ import fs from 'fs';
 import readline from 'readline';
 import Anonymizer from './anonymizer';
 import Splitter from './splitter';
-import { EncounterCollector } from './encounter_tools';
+import { EncounterCollector, TLUtilFunctions } from './encounter_tools';
 import ZoneId from '../../resources/zone_id';
 import argparse from 'argparse';
 import { LogUtilArgParse } from './arg_parser';
 
 // TODO: add options for not splitting / not anonymizing.
 const timelineParse = new LogUtilArgParse();
+const TFuncs = new TLUtilFunctions();
```

Comment:
Is there any state on `TLUtilFunctions`? Should all of its functions be static (e.g. regexes)?  And then you wouldn't need to make an object here, and could just refer to it off of the class itself.